packages/terra-core-docs/src/terra-dev-site/doc/form-input/About.1.doc.mdx

```diff
@@ -15,6 +15,7 @@ import UncontrolledDefaultInvalidExample from './example/uncontrolled/DefaultInv
 import UncontrolledDefaultIncompleteExample from './example/uncontrolled/DefaultIncompleteExample?dev-site-example';
 import UncontrolledInvalidWithIncompleteExample from './example/uncontrolled/InvalidWithIncompleteExample?dev-site-example';
 import UncontrolledDisabledInvalidExample from './example/uncontrolled/DisabledInvalidExample?dev-site-example';
+import PlaceholderExample from './example/uncontrolled/PlaceholderExample?dev-site-example';
```

Comment:
let's remove this examples from docs. As we don't want consumers to use this new MPage attribute to render placeholder.

Reply:
Removed the example.